collector/envoy/app/tracing/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM flask_service:python-3.10-slim-bullseye
 
-COPY --from=envoyproxy/envoy-dev:latest /usr/local/bin/envoy /usr/local/bin/envoy
+# envoy v1.23.0
+COPY --from=envoyproxy/envoy-dev:4e2c7adfb393a8ea2a38c94494ccd2ee0fd469f9 /usr/local/bin/envoy /usr/local/bin/envoy
```

Comment:
Do we benefit from pinning in these? I was thinking the drift will help monitor the status of integrations, but there are other ways.

Reply:
Agreed, removed :+1: